backoffice/backoffice/workflows/tests/cassettes/TestAuthorWorkflowViewSet.test_restart_with_params.yaml

```diff
@@ -0,0 +1,284 @@
+    headers:
+      Connection:
+      - close
+      Content-Length:
+      - '294'
+      Content-Type:
+      - application/problem+json
+      Date:
+      - Thu, 25 Jul 2024 15:29:21 GMT
+      Server:
+      - gunicorn
+      X-Robots-Tag:
+      - noindex, nofollow
+    status:
+      code: 404
+      message: NOT FOUND
+- request:
+    body: '{"dry_run": false, "dag_run_id": "fa7ca957-9589-4c19-8804-6b875b5ffbc5",
+      "reset_dag_runs": true}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Basic YWlyZmxvdzphaXJmbG93
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '96'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.31.0
+    method: GET
+    uri: http://host.docker.internal:8080/api/v1/dags/author_create_rejected_dag/dagRuns/fa7ca957-9589-4c19-8804-6b875b5ffbc5
+  response:
+    body:
+      string: "{\n  \"detail\": \"DAGRun with DAG ID: 'author_create_rejected_dag'
+        and DagRun ID: 'fa7ca957-9589-4c19-8804-6b875b5ffbc5' not found\",\n  \"status\":
+        404,\n  \"title\": \"DAGRun not found\",\n  \"type\": \"https://airflow.apache.org/docs/apache-airflow/2.8.3/stable-rest-api-ref.html#section/Errors/NotFound\"\n}\n"
+    headers:
+      Connection:
+      - close
+      Content-Length:
+      - '294'
+      Content-Type:
+      - application/problem+json
+      Date:
+      - Thu, 25 Jul 2024 15:29:21 GMT
+      Server:
+      - gunicorn
+      X-Robots-Tag:
+      - noindex, nofollow
+    status:
+      code: 404
+      message: NOT FOUND
+- request:
+    body: null
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Basic YWlyZmxvdzphaXJmbG93
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '0'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.31.0
+    method: DELETE
+    uri: http://host.docker.internal:8080/api/v1/dags/author_create_initialization_dag/dagRuns/fa7ca957-9589-4c19-8804-6b875b5ffbc5
+  response:
+    body:
+      string: ''
+    headers:
+      Connection:
+      - close
+      Content-Type:
+      - application/json
+      Date:
+      - Thu, 25 Jul 2024 15:29:22 GMT
+      Server:
+      - gunicorn
+      X-Robots-Tag:
+      - noindex, nofollow
+    status:
+      code: 204
+      message: NO CONTENT
+- request:
+    body: '{"dag_run_id": "fa7ca957-9589-4c19-8804-6b875b5ffbc5", "conf": {"workflow_id":
+      "fa7ca957-9589-4c19-8804-6b875b5ffbc5"}}'
+    headers:
+      Accept:
+      - '*/*'
+      Accept-Encoding:
+      - gzip, deflate
+      Authorization:
+      - Basic YWlyZmxvdzphaXJmbG93
+      Connection:
+      - keep-alive
+      Content-Length:
+      - '119'
+      Content-Type:
+      - application/json
+      User-Agent:
+      - python-requests/2.31.0
+    method: POST
```

Comment:
https://github.com/inspirehep/inspirehep/blob/master/backend/tests/conftest.py#L11-L36

we need to have similar config not to record internal services, such as opensearch

Reply:
now that I think about it backoffice tests are currently only accessing internal services since the `workflows` are now also contained in this repo

Reply:
the workflows despite it's in the same repo it's not the same service and for the moment we keep them separate